### ISSUE TITLE
projects/oblique/schematics dist fails on macOS due to cpx error

### ISSUE THREAD

**Issue by delixfe** [oblique-bit/oblique#26]
Putting the cpx globs in quotes seems to fix the issue:

```
"dist": "cpx 'index/**/*{.js,json}' '../../../dist/oblique/schematics/index' && cpx '{package.json,collection.json,migration.json}' '../../../dist/oblique/schematics && cpx index/ng-add/templates/*{.html,config} ../../../dist/oblique/schematics/index/ng-add/templates/'"
```

**Comment by gillerr** [comment]
Hi

Thanks for the reporting, I think putting globs in quotes breaks the dist on Windows, but I will try again.

**Comment by gillerr** [comment]
Works on Windows and Linux with the quotes. It have been included in the 6.1.5 hotfix
